desktop-app/app/components/WebView/index.js

```diff
@@ -990,9 +990,18 @@ class WebView extends Component {
           >
             <p>ERROR: {errorCode}</p>
             <p className={cx(styles.errorDesc)}>{errorDesc}</p>
+
             {proxyAuthError && (
               <p className={cx(styles.errorDesc)}>Proxy Authentication Error</p>
             )}
+
+            {errorCode <= -200 && errorCode >= -299 && (
```

Comment:
Can we have -200 and -299 externalized as a constant with a name that explains what it does? For eg: const `SSL_ERROR` ?

Reply:
On second thoughts, you can also wrap it in a function say, `isSslValidationFailed` and add that function to `responsively-app/desktop-app/app/utils/generalUtils.js`.

Reply:
Okay i will put this in a function inside generalUtils.js and add const for 200 and 299.

Reply:
done